src/main/java/core/basesyntax/io/impl/CsvReportSerializer.java

```
@@ -0,0 +1,39 @@
+package core.basesyntax.io.impl;
+
+import core.basesyntax.io.ReportSerializer;
+import core.basesyntax.model.FruitType;
+import core.basesyntax.model.Report;
+import java.io.BufferedWriter;
+import java.io.IOException;
+import java.math.BigInteger;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.List;
+
+public class CsvReportSerializer implements ReportSerializer {
+    @Override
+    public void save(Report report, Path path) {
+        List<String> csvRows = createCsvRows(report);
+        try (BufferedWriter writer = Files.newBufferedWriter(path)) {
+            for (String line : csvRows) {
+                writer.write(line);
+                writer.newLine();
+            }
+            writer.flush();
+        } catch (IOException e) {
+            throw new RuntimeException(e);
```

The catch block wraps IOException into a raw RuntimeException without context. This violates the checklist item: "Don't ignore exceptions in try-catch blocks". Rethrow with an informative message that includes the output `path` (for example: "Failed to write CSV report to '" + path + "': " + e.getMessage()) and include the original exception as the cause (new RuntimeException(message, e)). This will make errors actionable and meet the requirement to surface errors appropriately.